packages/ketcher-react/package.json

```diff
@@ -59,7 +59,7 @@
     "file-saver": "^2.0.2",
     "font-face-observer": "^1.0.0",
     "hoist-non-react-statics": "^3.3.2",
-    "intersection-observer": "^0.12.0",
+    "intersection-observer": "^0.12.2",
```

Comment:
Perhaps we don't need to have IntersectionObserver polyfill anymore as this API is now widely supported, but it's worth discussing separately